l10n/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  css: require('./css'),
+  svg: require('./svg'),
```

Comment:
Please remove the svg entry here, I just committed bc5c80ed773fc04ac46dbe2a59bb49a85886fc65 which removes the file. It was an oversight. We have not yet migrated over SVG data sets from our KumaScript repository.